client/blocks/comments/post-comment-content.jsx

```diff
@@ -48,6 +48,7 @@ class PostCommentContent extends React.Component {
 					<Emojify>
 						<div
 							className="comments__comment-content"
+							data-e2e-comment={ this.props.content.slice( 3, this.props.content.length - 5 ) }
```

Comment:
This string manipulation deserves some explanation: what does the code want to do?

Reply:
Hey @jsnajdr, thanks for asking about this one! I made a separate branch and PR for this change https://github.com/Automattic/wp-calypso/pull/27545 but made a mistake and create a branch for this PR from that one. I'll close first PR (since those changes are here) and copy comment:

_If I use only `this.props.content` then comment text has \<p> tag around it. Is there a better way to get content?_

Reply:
Is the comment guaranteed to always have `\n` at the end and be wrapped in `<p>` tag? It might be a good idea to check if they are present before stripping characters.

I think using `slice` is OK, but let's add some comments to the code that explain the magical "remove first 3 characters and last 5 characters". Is it removing `<p>` and `</p>\n`? Hard to know.

Reply:
Agree! I'll put those comments. 

For this case, comment always has `<p>` tag. [This](https://github.com/Automattic/wp-calypso/pull/27624#pullrequestreview-161989667)  is the where `\n` happens.